src/reports/library/cell.cpp

```diff
@@ -0,0 +1,48 @@
+#include <stdlib.h>
+#include <new>
```

Comment:
what is this needed for?

Reply:
it was used in previous versions, thanks!

Reply:
Please make sure that there aren't unnecessary includes in other files; I've think I've seen the `#include <new>` in a few places, possibly because of cut and paste.

Reply:
Sure, I am doing a header cleanup